src/Blazored.Toast/content/blazored-toast.css

```
@@ -25,6 +31,11 @@
     right: 2rem;
 }
 
+.position-bottomcenter {
+    bottom: 2rem;
+    left: 50%;
+    margin-left: -15em;
```

Nip picking here, but could this be updated to `rem` instead of `em` for consistency.
```suggestion
    margin-left: -15rem;
```